src/components/subscriptions/data/utils.js

```diff
@@ -0,0 +1,9 @@
+import { SEEN_SUBSCRIPTION_EXPIRATION_MODAL_COOKIE_PREFIX } from './constants';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getSubscriptionExpiringCookieName = ({
+  expirationThreshold, enterpriseId,
+}) => {
```

Comment:
It doesn't need to be, though if a function takes 2+ arguments in JS land, I typically pass it in as an object so that there's no confusion around which function argument serves what purpose. Using an object sort of mimics using kwargs vs. args in Python, where order of the argument doesn't matter and it's more explicit that some value is for a given argument.

Example: 

The function `myAwesomeFunction(boolA, boolB) { ... }` would be used as `myAwesomeFunction(true, false)` but I don't know which true/false value goes to which argument until I read the implementation of `myAwesomeFunction`, and may easily unintentionally flip the ordering of the arguments.

If instead the function were `myAwesomeFunction({ boolA, boolB })`, it'd be used as `myAwesomeFuction({ boolA: true, boolB: false })` which is much more understandable at a quick glance IMO. It's also nice that the ordering of the arguments don't matter.